client/src/state-machine.ts

```diff
@@ -179,8 +170,9 @@ async function getSearchResult(addr: SearchAddressWithoutBbl): Promise<WowState>
     };
   } else if (apiResults.addrs.length === 0) {
     const buildingInfoResults = await APIClient.getBuildingInfo(apiResults.geosearch.bbl);
-    const nychaData = getNychaData(apiResults.geosearch.bbl);
     const buildingInfo = buildingInfoResults.result[0];
+    const nychaStatsResults = await APIClient.getNychaStats(apiResults.geosearch.bbl);
```

Comment:
Hmm, does this effectively mean that we're making two separate API calls to the server for every building a user looks up?  I'm wondering if we actually want to just add NYCHA info to the `getBuildingInfo()` API endpoint instead, but that's potentially a premature optimization... Might be worth filing an issue to consider combining the two endpoints in the future, though?

Also note that because this code consists of two _sequential_ `await` calls, this effectively means that we're making one request, waiting for the result, and then making the other request, which could be slower than e.g. using [`Promise.all()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) to run them in parallel, which could be faster (although it depends on how our server processes requests too).  Again, probably not a big deal in the grand scheme of things, but it might be worth doing a bit of measurement up-front using e.g. the [Chrome network panel](https://developer.chrome.com/docs/devtools/network/) to see if/how performance is affected.

Reply:
@toolness I was inspired by this comment and decided to include the nycha stats SQL query as part of the building info query so we only need to make one network request. Do you mind reviewing my work once more?